server/spec/unit/controllers/Cards/completion.hook.js

```diff
@@ -0,0 +1,9 @@
+import { before } from 'hooks'
+
+before("Card Completion Collection > POST", (transaction) => {
+  transaction.skip = true
```

Comment:
@datarockets/cuties is this some kind of stub for db requests?

Reply:
@DDKatch it skips test suite, [see](https://dredd.org/en/latest/hooks-nodejs.html#how-to-skip-tests)